### ISSUE TITLE
Enhancement: Ability to create bookmarks

### ISSUE THREAD

**Issue by elpraga** [owncloud/android#1407]
I think that the current concept of Favorites is somewhat misleading. I might have a "Favorite" folder that lies deep within the folder tree which I want to access quickly, but it does not necessarily mean that I want this whole folder synced on my phone. 

Would it be possible to make "Bookmarks" in the left menu part (where the access to configuration, and on device / all files switch is now) that would be merely a list of shortcuts to get us quickly deep into menu structure?

**Comment by tobiasKaminsky** [comment]
@jancborchardt 
Please help me to recall what (and if) we discussed on this topic:
- "Keep in sync" (currently Favorite) should get a new name?
- In the navigation drawer a new entry "Favorite" should list all files/folders that are "starred"? (Just like @elpraga is suggesting)

Is this correct?

**Comment by elpraga** [comment]
My original idea approached two things:
* now "Favorite" is not favorite as in browser or file manager manner, but rather "always synced", a new name You mention could actually be less confusing
* It might be nice to have Bookmarks entry in the drawer as in a web browser or file manager style

**Comment by jancborchardt** [comment]
Thing is, this introduces a new layer of complexity. Sorry to bring it up, but Dropbox does it just like that: favs mean that on mobile the files are made available offline. And we do not even sync the favorite state of a file between server and mobile apps right now.

So we definitely have some details to work out and polish before we can think of adding more stuff. Hope that's understandable.

**Comment by elpraga** [comment]
I was only trying to point out a possible enhancement :) I suppose You are quite busy, and I definitely understand that, I wish I could add some code as well, and not just ideas. 

Please, note that I was not suggesting to sync the Bookmarks (as in filemanager style) with the server, I think, there is little need for that. I just thought about it when going up and down the directory tree to get somewhere.  

I think the OC app is actually an Android file manager on steroids, and bookmarks are quite common feature in them.

I am sorry, but I cannot resist to rephrase You..
I'd love to help to make the OC Android app better than Dropbox app! :D

Thank all You coders a lot for Your work!!

**Comment by elpraga** [comment]
OC 9.0 implemented tags on files and folders. If implementing somethings, tags make much more sense, as they are supported by core.

Closing as obsolete.

**Comment by AndyScherzinger** [comment]
@elpraga but then tag would still need to be supported by the Android client in order for you to use it (!)

**Comment by elpraga** [comment]
Sure thing @AndyScherzinger , what I ment was that if something should be implemented in the android app, it should be the new tag system that is supported by core, rather than a separate bookmark mechanism, as both provide very similar functionality.

**Comment by AndyScherzinger** [comment]
Yeah, makes sense :) In this case you could open up a feature request if you want to :+1:
